recipes-pv/pantavisor/pantavisor_git.bb

```diff
@@ -46,6 +46,9 @@ EXTRA_OECMAKE += "${@bb.utils.contains('DISTRO_FEATURES', 'usrmerge', '-DPANTAVI
 EXTRA_OECMAKE += "${@bb.utils.contains('PANTAVISOR_FEATURES', 'debug', '-DPANTAVISOR_DEBUG=ON', '', d)}"
 EXTRA_OECMAKE += "${@bb.utils.contains('PANTAVISOR_FEATURES', 'dm-crypt', '-DPANTAVISOR_DM_CRYPT=ON', '', d)}"
 EXTRA_OECMAKE += "${@bb.utils.contains('PANTAVISOR_FEATURES', 'dm-verity', '-DPANTAVISOR_DM_VERITY=ON', '', d)}"
+EXTRA_OECMAKE += "${@bb.utils.contains('PANTAVISOR_FEATURES', 'pvtx', '-DPANTAVISOR_PVTX=ON', '', d)}"
+EXTRA_OECMAKE += "${@bb.utils.contains('PANTAVISOR_FEATURES', 'pvtx-bz2', '-DPANTAVISOR_PVTX_BZ2=ON', '', d)}"
+EXTRA_OECMAKE += "${@bb.utils.contains('PANTAVISOR_FEATURES', 'pvtx-static-libgcc', '-DPANTAVISOR_PVTX_STATIC_LIBGCC=ON', '', d)}"
```

Comment:
lets always build pvtx-static and put that in a separate package called 'pvtx-static'

lets also always build pvtx and put it in a package pvtx

lets keep the pvtx-bz2 as a PANTAVISOR_FEATURE that influences dependencies and both pvtx packages

Reply:
ok